Replaced "the" article with "a". Replaced the past tense with the present tense. (#9481)
mdn/content: files/en-us/web/http/headers/last-modified/index.md

---
{{HTTPSidebar}}

The **`Last-Modified`** response HTTP header contains a date
and time when the origin server believes the resource was last modified. It is used

> as a validator to determine if a resource receives or stores the same. Less accurate

Reviewer: This seems to change a bit the meaning. What about
```suggestion
as a validator to determine if the resource is the same as the previously stored one. Less accurate
```

(Feel free to iterate here)

Reviewer 2: yeap, I did not get the meaning. This one "as a validator to determine if the resource is the same as the previously stored one" is correct. Thanks)
